### PR TITLE
Player-4830 CompleteSampleApp Crash Fix

### PR DESCRIPTION
**Pull request by skumar2905**
Player-4830 Fixed CompleteSampleApp crash issue on selecting "Asset Options".  Added "CustomActivity" to AndroidManifest.xml.
Also Fixed AUTOPLAY. Now works as per selection.

### CODE REVIEW

**Review by DmitriiMaslov — CHANGES_REQUESTED** [review]
Please add clear description on what was done in this PR. Follow this page - https://wiki.corp.ooyala.com/display/ENG/Pull+Requests+Review+Culture

File: CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
   String PCODE = null;
   String DOMAIN = null;
   private final String ACCOUNT_ID = "accountID";
+  Boolean autoPlay = false;
```
Comment (isheftel-ooyala) [inline CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java]: Seems like you have some other or outdated version.
There is already an autoplay field in latest candidate.
image

Comment (skumar2905) [reply CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java]: @isheftel-ooyala  I don't see AUTOPLAY in below. Please recheck. https://github.com/ooyala/android-sample-apps/blob/candidate/CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java

**Comment by skumar2905** [comment]
@DmitriiMaslov Updated description.

File: CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java (hunk 1 above)
Comment (isheftel-ooyala) [reply CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java]: Ah, that's another Activity, ok

File: CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java [hunk 2]
```diff
@@ -43,6 +43,7 @@
   String PCODE = null;
   String DOMAIN = null;
   private final String ACCOUNT_ID = "accountID";
+  Boolean AUTOPLAY = false;
```
Comment (isheftel-ooyala) [inline CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java]: it can be a primitive + private modifier

Comment (skumar2905) [reply CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java]: @isheftel-ooyala Done

**Review by SergeyBicarte — APPROVED** [review]
LGTM , other side /android-sample-apps/ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java should be aligned in same way then

**Review by isheftel-ooyala — APPROVED** [review]
LGTM

**Pull request merged**